api/src/services/validation-service.ts

```diff
@@ -445,6 +445,7 @@ export class ValidationService extends DBService {
     defaultLog.debug({
       label: 'persistTransformationResults',
       submissionId,
+      fileBuffers,
```

Comment:
I had removed this parameter in a previous commit because the output becomes so huge that it truncates any logs before this one. I suppose this could just be commented out when applicable

Reply:
For sure, not sure how it returned. odd. WIll remove